asv/commands/find.py

```diff
@@ -128,10 +135,29 @@ def do_benchmark(i):
 
             env.install_project(conf, repo, commit_hash)
 
+            params = dict(machine_params.__dict__)
+            params['python'] = env.python
+            params.update(env.requirements)
+
+            result = Results(
+                params,
+                env.requirements,
+                commit_hash,
+                repo.get_date(commit_hash),
+                env.python,
+                env.name)
+
+            if save_results:
+                result.load_data(conf.results_dir)
```

Comment:
Here we are loading result to avoid overwriting other valid data, right ?. Can you do add a comment?